code/controllers/subsystem/shuttles.dm

```diff
@@ -37,7 +37,6 @@ SUBSYSTEM_DEF(shuttle)
 	var/list/shoppinglist = list()
 	var/list/requestlist = list()
 	var/list/supply_packs = list()
-	var/datum/round_event/shuttle_loan/shuttle_loan
```

Comment:
How the hell is this passing CI when we dont have this type?


I am mystified.

Reply:
That's sort of what I would have expected, too. Perhaps it's not a case that dreamchecker checks/cares about?

Reply:
It does look like byond just ignores this sort of thing entirely for whatever reason, no errors or anything. Definitely should be removed from here though either way. 👍
![image](https://user-images.githubusercontent.com/57483089/129114182-e8c42ecd-87fb-4855-bbc6-1b0559e60d86.png)